charts/kubewarden-controller/Chart.yaml

```diff
@@ -13,7 +18,34 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.1
+version: 1.1.10
```

Comment:
I do not think this is an issue. But I'm curious why the jump from `1` to `10`. Is there some requirement for that? I read the RFC but I cannot remember nothing that requires that.

Reply:
Well, I wanted to leave some room in case we had a bump or two of kubewarden-defaults, which is now at 1.1.7. And it also seemed like a round number to be easy to remember if a chart has Rancher support or not :man_shrugging:  :sweat_smile:

Reply:
I would bump to `1.2.0`. We can use this version as the baseline for all the helm charts that are inside of this PR.

> Well, I wanted to leave some room in case we had a bump or two of kubewarden-defaults, which is now at 1.1.7.

What do you mean with that?

Reply:
Bumped to `1.2.0` in amended commit https://github.com/kubewarden/helm-charts/pull/129/commits/c684e4f2e778d0725cad013bc4fcb429417f74e8.

> What do you mean with that?

I was just thinking that there may have been other chart releases meanwhile this PR got merged, so I left unused version numbers as a buffer.